### PR TITLE
IDCORE-4594: add wallet.source parameter to POST and PUT requests

### PR DESCRIPTION
**Pull request by gasper-vrhovsek**
Accept wallet source as a parameter and propagate it to wallet settings
dict.

Signed-off-by: Gasper Vrhovsek <gasper.vrhovsek@gmail.com>

### CODE REVIEW

**Review by dejsenlitro — DISMISSED** [review]

**Review by urossmerdelj — APPROVED** [review]

File: aries_cloudagent/multitenant/admin/tests/test_routes.py [hunk 1]
```diff
@@ -206,6 +208,7 @@ async def test_wallet_create_optional_default_fields(self):
         body = {
             "wallet_name": "test",
             "wallet_key": "test",
+            "wallet_source": "",
```
Comment (acuderman) [inline aries_cloudagent/multitenant/admin/tests/test_routes.py:211]:
```suggestion
            "wallet_source": "wallet_source_test",
```

Comment (gasper-vrhovsek) [reply aries_cloudagent/multitenant/admin/tests/test_routes.py:211]: As i understood this test, it was for default values, which should be empty for `wallet_source`?

File: aries_cloudagent/multitenant/admin/routes.py [hunk 2]
```diff
@@ -98,6 +98,12 @@ class CreateWalletRequestSchema(OpenAPISchema):
         example="https://aries.ca/images/sample.png",
     )
 
+    wallet_source = fields.Str(
+        description="Wallet source. Arbitrary value describing the wallet creation source",
+        example="MyNewWalletSource",
```
[the remark this answers is not in the document]
Comment (gasper-vrhovsek) [reply aries_cloudagent/multitenant/admin/routes.py]: done

File: aries_cloudagent/multitenant/admin/tests/test_routes.py (hunk 1 above)
Comment (acuderman) [reply aries_cloudagent/multitenant/admin/tests/test_routes.py:211]: Shouldn't `None` be a default value? :thinking:

Comment (gasper-vrhovsek) [reply aries_cloudagent/multitenant/admin/tests/test_routes.py:211]: Yes, probably should be :)

Comment (gasper-vrhovsek) [reply aries_cloudagent/multitenant/admin/tests/test_routes.py]: Changed to `None`

**Review by acuderman — APPROVED** [review]

File: aries_cloudagent/multitenant/admin/tests/test_routes.py (hunk 1 above)
Comment (gasper-vrhovsek) [reply aries_cloudagent/multitenant/admin/tests/test_routes.py]: Fixed, set `None` as default value.

**Pull request merged**